config/crd/bases/ran.openshift.io_clustergroupupgrades.yaml

```diff
@@ -37,6 +37,10 @@ spec:
           spec:
             description: ClusterGroupUpgradeSpec defines the desired state of ClusterGroupUpgrade
             properties:
+              clusterSelector:
```

Comment:
Including a description here of the structure/format of the string would be really helpful.